src/main/webapp/css/srp_style.css

```diff
@@ -22,6 +22,19 @@
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  height: 260px;
+  overflow: scroll;
+}
+
+.card::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+  background-color: WHITE;
+}
```

Comment:
Add a visible scroll bar to each card to signal to the user that the card it's scrollable and not just weirdly cutoff.
```
.card::-webkit-scrollbar-thumb {
  background: lightgrey;
  border-radius: 10px;
}
```
Optionally, maybe add hover styling? At least browser scrollbars on my laptop have hover styling.
```
.card::-webkit-scrollbar-thumb:hover {
  background: grey;
  border-radius: 10px;
}
```